src/command_modules/azure-cli-interactive/azclishell/az_completer.py

```
@@ -242,7 +243,7 @@ def gen_dynamic_completions(self, text):
                                 pass  # other completion method used
 
         # if the user isn't logged in
-        except (CLIError, AttributeError, ValueError):  # service client throws
+        except (CLIError, AttributeError, ValueError, ValidationError):  # service client throws
```

If there's a reason why you don't want to catch all exceptions, please comment.